app/src/main/res/values/strings.xml

```diff
@@ -601,6 +601,12 @@
     <string name="choose_instance_prompt">Choose an instance</string>
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
+    <string name="remove_watched">Remove watched</string>
+    <string name="remove_watched_popup_title">Remove watched videos?</string>
+    <string name="remove_watched_popup_warning">"Videos that have been watched\nbefore and after being added to the playlist will be removed.\nAre you sure? This cannot be undone!</string>
+    <string name="remove_watched_popup_yes">Yes</string>
+    <string name="remove_watched_popup_cancel">Cancel</string>
```

Comment:
You shouldn't add two new strings: "Yes" and "Cancel" either already exist in `strings.xml` (search with CTRL+F if that's the case) or even are in the default android system strings.

Reply:
There is a reason we created these strings (yes no cancel ok…) and we aren't using android ones: Android isn't translated to all languages, I can think of Esperanto specifically.

Reply:
Fixed in db335d5